src/ploneintranet/workspace/browser/templates/content_macros.pt

```diff
@@ -44,7 +89,7 @@
                     </fieldset>
 
                     <label class="timezone">Timezone
-                        <select class="timezone" tal:attributes="disabled read_only" >
+                        <select class="timezone">
                             <option data-timezone-id="80" gmt-adjustment="GMT+12:00" data-use-daylight-time="1" value="12">(GMT+12:00) Auckland, Wellington</option>
```

Comment:
Not consciously. I'd probably like to blame that on a merge mistake. We don't need or use it, so we can disable it again. @cillianderoiste agree?